From d6ed681d64f864dd1df5043e88585bc284c80cd8 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 28 Nov 2017 22:25:07 +0800
Subject: [PATCH 2930/5242] MLK-16604-3 dt-bindings: usb: xhci: add
 usb3-resume-missing-cas property

commit  cfcd47ee5b9cd6facdb7a7d0c3227a7676686f63 from
https://source.codeaurora.org/external/imx/linux-imx.git

There is already one quirk for usb3 xhci flag XHCI_MISSING_CAS, for
those platform with OF we can use usb3-resume-missing-cas to enable
this quirk to work around usb3 resume from system sleep.

Signed-off-by: Li Jun <jun.li@nxp.com>
Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/devicetree/bindings/usb/usb-xhci.txt |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/usb-xhci.txt b/Documentation/devicetree/bindings/usb/usb-xhci.txt
index bd1dd31..b7a31bc 100644
--- a/Documentation/devicetree/bindings/usb/usb-xhci.txt
+++ b/Documentation/devicetree/bindings/usb/usb-xhci.txt
@@ -35,6 +35,8 @@ Optional properties:
   - usb2-lpm-disable: indicate if we don't want to enable USB2 HW LPM
   - usb3-lpm-capable: determines if platform is USB3 LPM capable
   - quirk-broken-port-ped: set if the controller has broken port disable mechanism
+  - usb3-resume-missing-cas: set if the CAS(Cold Attach Status) may lose in case
+    device plugged in while system sleep.
   - imod-interval-ns: default interrupt moderation interval is 5000ns
   - phys : see usb-hcd.txt in the current directory
 
-- 
1.7.9.5

